docs/getting_started/basics/batch.md

```diff
@@ -0,0 +1,35 @@
+# Batch Execution
+
+Shift is able to run workflows without the need of a UI interface. This is called Batch mode and can be used via the Shift Python API.
+
+## Opening a Workflow
+
+The following snippet of code can be used to open a Shift workflow file:
+
+```python
+from shift.core import files
+
+workflow, catalog_manifest = files.openBatchWorkflow("<path_to_your_workflow_file>")
+```
+
+The execution of this method will return the `SWorkflow` Python class containing the workflow and a Python dictionary containing the information on the Python modules loaded by the workflow. Have a look at the [Shift API](../../reference/api.md) for more information on the method and the `SWorkflow` class.
```

Comment:
@juliavisserinbibo should consider the catalog_manifest returned like a dict of "Python modules loaded" by Shift, or a dict of "Shift Catalogs"? I know that a catalog is a pyModule at the end, but just to be sure how to refer to the catalog in the documentation.

Reply:
Right, it should be
```suggestion
The execution of this method will return the `SWorkflow` Python class containing the workflow and a Python dictionary containing the information on the Shift catalogs loaded by the workflow. Have a look at the [Shift API](../../reference/api.md) for more information on the method and the `SWorkflow` class.
```

Reply:
right! done!

Reply:
I think you have a typo, pls check to fix "Shft catalogs" by Shift catalogs @albertochimentiinbibo

Reply:
great catch man thanks a lot!